### PR TITLE
Add NavBar, Form Columns, Map Placeholder, and Widget Cards

### LINKED ISSUE #43: Create initial HTML site layout with Headers and blocks where each widget will go

### PR DESCRIPTION
**Pull request by carsondrobe**
This pull request includes the following changes:

- Added NavBar and Form Columns

- Added map placeholder

- Added widget cards

### CODE REVIEW

**Comment by carsondrobe** [comment]
Merge conflict with db.sqlite3 and I have no idea how to resolve because this is a binary file

**Review by Davis245 — APPROVED** [review]
Looks like a good outline for the dashboard! I noticed this does not include the second page; however, including the second page in this issue would make it quite big, so we should create a new issue for the second page.

**Comment by carsondrobe** [comment]
Yes I saw there is another issue for the second page: https://github.com/carsondrobe/Fellas/issues/44

File: Site/bc_weather_station_dashboard/bc_weather_station_dashboard/__pycache__/__init__.cpython-311.pyc
Comment (robertyac) [inline Site/bc_weather_station_dashboard/bc_weather_station_dashboard/__pycache__/__init__.cpython-311.pyc:1]: If possible in the future any .pyc don't need to be uploaded as they are just temporary files that get generated when needed. Even though the .pyc is in the git ignore hopefully in the future now it won't upload these files.

**Review by robertyac — APPROVED** [review]
Good initial outline for our front end page 1 looks great. We'll continue to build on this.

File: Site/bc_weather_station_dashboard/bc_weather_station_dashboard/__pycache__/__init__.cpython-311.pyc
Comment (carsondrobe) [reply Site/bc_weather_station_dashboard/bc_weather_station_dashboard/__pycache__/__init__.cpython-311.pyc:1]: Yeah I added it to gitignore but its still committing them. Not sure why I'll look into it

**Pull request merged**